components/summary-list/src/SummaryList.tsx

```diff
@@ -11,9 +11,18 @@ export type SummaryListProps = SummaryListContainerProps & {
 
 const SummaryListComponent: FC<SummaryListProps> = ({
   classBlock = 'govuk-summary-list',
-  items,
+  items: _items,
   ...props
 }) => {
+  const items = _items.map((item) => {
+    if (!Array.isArray(item.actions)) {
+      typeof item.classModifiers === 'string' ? item.classModifiers = [item.classModifiers] : item.classModifiers = [];
+      item.classModifiers.push('no-actions');
+    }
+
+    return item
+  })
```

Comment:
I think there is a slight problem here, in that `no-actions` will be pushed even when when none of the items have actions.

That's probably not harmful, but it's not what GDS do.

I can just patch that up though...